package.json

```
@@ -47,6 +47,9 @@
     "typescript": "^4.4.2"
   },
   "dependencies": {
+    "@codemirror/language": "^6.0.0",
```

It doesn't matter in this case, but here it should use Licat's fork because of additional custom exports. I overlooked that when I wrote that three weeks ago.
```suggestion
    "@codemirror/language": "https://github.com/lishid/cm-language",
```